sentry/src/main/java/io/sentry/util/Platform.java

```diff
@@ -0,0 +1,19 @@
+package io.sentry.util;
+
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public final class Platform {
+  // All system properties on Android:
+  // https://developer.android.com/reference/java/lang/System#getProperties()
+  private static final boolean isAndroid =
+      "The Android Project".equals(System.getProperty("java.vendor"));
```

Comment:
it could throw `SecurityException` or `IllegalArgumentException`, if so, SDK will crash the App, should we handle such a use case?

Reply:
Never seen it happening in wild, but better safe than sorry. Exceptions are now caught.